samples/apps/forum/src/types/ccf.ts

```diff
@@ -61,6 +81,11 @@ export interface AESKWPParams extends WrapAlgoParams {
   name: "AES-KWP";
 }
 
+export interface RsaOaepAESKWPParams extends WrapAlgoParams {
```

Comment:
I'd say either RSAOAEPAESKWParams or RsaOaepAesKwpParams, but this is a ConfusingWAYToDOIT.

Reply:
Yep, that part slipped through; entirely my fault :-)

Reply:
Not related to this PR, this was just an inconsistency between the two ccf.ts copies.